flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -1121,6 +1122,35 @@ public void testSetCommand() throws Exception {
         assertThat(result.toString()).isEqualTo(String.format("[+I[%s]]", path));
     }
 
+    @Test
+    public void testAddCommand() {
+        TableEnvironmentInternal tableEnvInternal = (TableEnvironmentInternal) tableEnv;
+        Parser parser = tableEnvInternal.getParser();
+
+        // test add jar
+        Operation operation = parser.parse("add jar test.jar").get(0);
+        assertThat(operation).isInstanceOf(AddJarOperation.class);
+        assertThat(((AddJarOperation) operation).getPath()).isEqualTo("test.jar");
+        // test add jar with variable substitute
+        operation = parser.parse("add jar \"${hiveconf:common-key}.jar\"").get(0);
+        assertThat(operation).isInstanceOf(AddJarOperation.class);
+        assertThat(((AddJarOperation) operation).getPath()).isEqualTo("\"common-val.jar\"");
+
+        // test unsupported add command
+        assertThatThrownBy(() -> tableEnv.executeSql("add jar t1.jar t2.jar"))
+                .isInstanceOf(UnsupportedOperationException.class)
+                .hasMessage(
+                        "Only support add one jar in one single statement. Usage: add JAR <value>");
+        assertThatThrownBy(() -> tableEnv.executeSql("add File t1.txt"))
+                .isInstanceOf(UnsupportedOperationException.class)
+                .hasMessage(
+                        "Only support add one jar in one single statement. Usage: add JAR <value>");
+        assertThatThrownBy(() -> tableEnv.executeSql("add Archive t1.tgz"))
```

Comment:
throw a specific exception for the ADD FILE and ADD ARCHIVE is not supported yet?